docs/how-to-guides/setup-thirdparty-auth-for-grafana.md

```diff
@@ -0,0 +1,144 @@
+
+- Set **Authorization callback URL** to `https://grafana.<cluster name>.<DNS zone>/login/github`.
+
+- Make a note of `Client ID` and `Client Secret`, they will be needed in [Step
+  3](#step-3-add-secret-information).
+
+### Step 2: Add `prometheus-operator` component configuration
+
+Create a file named `prometheus-operator.lokocfg` with the following contents or if you already
+have `prometheus-operator` installed then add the following contents to the existing configuration:
+
+```tf
+variable "gf_auth_github_client_id" {}
+variable "gf_auth_github_client_secret" {}
+variable "gf_auth_github_allowed_orgs" {}
+
+component "prometheus-operator" {
+  grafana {
+    secret_env = {
+      "GF_AUTH_GITHUB_ENABLED"               = "'true'"
+      "GF_AUTH_GITHUB_ALLOW_SIGN_UP"         = "'true'"
+      "GF_AUTH_GITHUB_SCOPES"                = "user:email,read:org"
+      "GF_AUTH_GITHUB_AUTH_URL"              = "https://github.com/login/oauth/authorize"
+      "GF_AUTH_GITHUB_TOKEN_URL"             = "https://github.com/login/oauth/access_token"
+      "GF_AUTH_GITHUB_API_URL"               = "https://api.github.com/user"
+      "GF_AUTH_GITHUB_CLIENT_ID"             = var.gf_auth_github_client_id
+      "GF_AUTH_GITHUB_CLIENT_SECRET"         = var.gf_auth_github_client_secret
+      "GF_AUTH_GITHUB_ALLOWED_ORGANIZATIONS" = var.gf_auth_github_allowed_orgs
+    }
+
+    ingress {
+      host = "grafana.<cluster name>.<DNS zone>"
+    }
+  }
+}
+```
+
+> **NOTE**: On Packet, you either need to create a DNS entry for `grafana.<cluster name>.<DNS zone>`
+> and point it to the Packet external IP for the contour service (see the [Packet ingress guide for
+> more details](./ingress-with-contour-metallb.md)) or use the [External DNS
+> component](../configuration-reference/components/external-dns.md).
+
+> **NOTE**: In the above configuration, boolean values are set to `"'true'"` instead of bare
+> `"true"` because Kubernetes expects the key-value pair to be of type `map[string]string` and not
+> `map[string]bool`.
+
+### Step 3: Add secret information
+
+Create a `lokofg.vars` file or add the following to an existing file, setting the values of this
+secret as needed:
+
+```tf
+gf_auth_github_client_id     = "YOUR_GITHUB_APP_CLIENT_ID"
+gf_auth_github_client_secret = "YOUR_GITHUB_APP_CLIENT_SECRET"
+gf_auth_github_allowed_orgs  = "YOUR_GITHUB_ALLOWED_ORGANIZATIONS"
+```
+
+Replace `YOUR_GITHUB_APP_CLIENT_ID` with `Client ID` and `YOUR_GITHUB_APP_CLIENT_SECRET` with
+`Client Secret` collected in [Step 1](#step-1-create-github-application). And replace
+`YOUR_GITHUB_ALLOWED_ORGANIZATIONS` with Github organization that your users belong to.
+
+### Step 4: Deploy and access the dashboard
+
+Deploy the `prometheus-operator` component using the following command:
+
+```bash
+lokoctl component apply prometheus-operator
+```
+
+Go to `https://grafana.<cluster name>.<DNS zone>` and use the newly appeared **Sign in with GitHub**
```

Comment:
I don't think we need to mention "newly appeared" as it seems distracting for the user.

```suggestion
Go to `https://grafana.<cluster name>.<DNS zone>` and use the **Sign in with GitHub**
```